.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+onlyLabels: 
+  - "needs info"
```

Comment:
Nice! I wish we'd use that on server

Reply:
Indeed. I copied it over from Text :relaxed: